scripts/build.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/bash
```

Comment:
Should this script should just be changed to wrap the poetry build?

Reply:
Will explain in a minute